webserver_test.py

```diff
@@ -51,11 +62,11 @@ def outputChecker(curl, curl_expected):
 
 # Request an echo from the webserver using curl
 curl = Popen(["curl", "-0", "-s", "localhost:2020/echo"], stdout=PIPE)
-ec = outputChecker(curl, curl_output_expected_echo)
+ec += outputChecker(curl, curl_output_expected_echo)
 
 # Request an echo from the webserver using curl
 curl = Popen(["curl", "-0", "-s", "localhost:2020/echo/"], stdout=PIPE)
-ec = outputChecker(curl, curl_output_expected_echo2)
+ec += outputChecker(curl, curl_output_expected_echo2)
```

Comment:
rationale behind this:
Before it was just setting the output to ec (it was "=" instead of "+=". This would overwrite the previous error results, so technically the previous test could have failed but we would  not know.